tests/test_integration.py

```diff
@@ -1315,6 +1315,33 @@ def test_no_emit_warnings_verbose_override():
   assert stderr
 
 
+def test_undeclared_setuptools_import_on_pex_path():
```

Comment:
Consider adding a note about what this test means. For example, link to the relevant issue.

Reply:
Done!